charts/liv-chat/templates/deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   labels:
     draft: {{ default "draft-app" .Values.draft }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+{{- if .Values.cluster.scService.podLabels }}
+{{ toYaml .Values.cluster.scService.podLabels | indent 8 }}
+{{- end }}
```

Comment:
Do we need these labels in `scc-broker-deployment` and `scc-state-deployment` files as well?

Reply:
yes, I have added labels to these files as well.